src/NHSD.BuyingCatalogue.Identity.UserDatabase/Deployment/CreateTestUsers.sql

```diff
@@ -1,38 +1,42 @@
-﻿IF NOT EXISTS(
+﻿DECLARE @aliceEmail AS nvarchar(50) = N'AliceSmith@email.com';
+DECLARE @bobEmail AS nvarchar(50) = N'BobSmith@email.com';
+
+IF NOT EXISTS(
   SELECT *
   FROM dbo.AspNetUsers
-  WHERE UserName IN ('alice', 'bob'))
+  WHERE UserName IN (@aliceEmail, @bobEmail))
 BEGIN
 	DECLARE @address AS nchar(108) = N'{ "street_address": "One Hacker Way", "locality": "Heidelberg", "postal_code": 69118, "country": "Germany" }';
 
 	DECLARE @aliceId AS nchar(36) = CAST(NEWID() AS nchar(36));
 	DECLARE @bobId AS nchar(36) = CAST(NEWID() AS nchar(36));
 
-	-- 'Pass123$'
+	DECLARE @aliceNormalizedEmail AS nvarchar(50) = N'ALICESMITH@EMAIL.COM';
+    DECLARE @bobNormalizedEmail AS nvarchar(50) = N'BOBSMITH@EMAIL.COM';
+
+    -- 'Pass123$'
 	DECLARE @alicePassword AS nvarchar(200) = N'AQAAAAEAACcQAAAAEFSsEthAqGVBLj1P1gF9puxtXm18lKHlmuh9J/Ib0KKBO3GjQvxymJbzpSqy0zuOHg==';
 
 	-- 'Pass123$'
 	DECLARE @bobPassword AS nvarchar(200) = N'AQAAAAEAACcQAAAAEOzr1Zwpoo1pKsTa+S65mBZVG4GIy6IYH/IAED6TvBA+FIMg8u/xb0b/cfexV7SHNw==';
 
-	INSERT INTO dbo.AspNetUsers(Id, AccessFailedCount, ConcurrencyStamp, EmailConfirmed, LockoutEnabled,
+	INSERT INTO dbo.AspNetUsers(Id, Email, NormalizedEmail, AccessFailedCount, ConcurrencyStamp, EmailConfirmed, LockoutEnabled,
```

Comment:
Very noddy but one suggestion here is to group the UserName & NormalizedUserName fields together along with the Email & NormalizedEmail.